bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54.html

```diff
@@ -122,5 +149,13 @@ <h4 class="up-to-date"><span>{{ _('Your Firefox is up to date.') }}</span></h4>
 {% block email_form %}{% endblock %}
 
 {% block js %}
-  {% javascript 'firefox_whatsnew_54' %}
+  {% if variation == 'c' %}
```

Comment:
I don't suggest changing anything as this is all working, but fwiw I find all this logic in a single template a bit hard to follow at times. This is probably just personal preference, but I find separate template variations feel a bit cleaner.